### DISCUSSION TITLE
Keep this repo release version in step with main ocular repo

### DISCUSSION THREAD

**Discussion by slfhstr** [simonwep/ocular#46] [Ideas]
Currently the ocular docker latest release is 1.20 on May 28.
Tne main repo is shown as 1.4 on May 1st.
This creates the impression that ocular-docker is behind the main repo.
Maybe this is true but I suspect not.
It would be worth keeping the release versions in line, even by frigging it with a couple of fake releases so both are on v1.4 (or whatever in future).

**Comment by simonwep** [comment]
This is on purpose as `ocular-docker` is a composition of `genesis` and `ocular`, a release of `ocular-docker` does not necessarily mean a new feature in `ocular`. But I can see how this might be confusing for others... What would you propose in the situation `ocular` is at v1.5, `ocular-docker` at v1.5 as well but there's an update in `genesis`, the backend, which requires a new release of ocular-docker?

I'm also thinking of merging both the `ocular-docker` repo and `ocular`. But I want to avoid having ya'll have to curl the whole source-code even though it's all in docker.

I'm definitely open for ideas here if you have any :)

**Comment by slfhstr** [comment]
Thanks for taking the time to reply and explain.

You make good points, I didn't consider the `genesis` issues.
Given those considerations, my suggestion doesn't work well, sorry, maybe better to ditch it.
I think the only solution in practice is to be explicit in terms of titles or descriptions, e.g. `ocular-docker v1.2 packaging ocular v1.4` or whatever.

About separate repos, it seems a bit more traditional to have it all in one repo, but actually I like your approach, it certainly makes it a lot easier to deploy `ocular-docker`.  Given that docker deployment  benefits are at least partially about not worry about building source code and sorting out dependencies, from a selfish personal viewpoint, it's nice to have a separate repo for it.  But I guess it might be more work for you.

Here also the solution might be about documentation in a single repo, e.g. :
"For docker deployment, no need to clone the whole repo, just download the contents of the docker-compose folder and follow the instructions in the README.md in that folder."  

Sadly no easy clear answer, other than ... what makes your life easier (as you are the dev and all this is probably a nuisance set against the actual dev work) ?

**Comment by simonwep** [comment]
Hey, just wanted to give you a heads up that I'll have another look into this because of https://github.com/simonwep/ocular-docker/issues/5 - e.g. providing a single docker image in this repository / deprecating the other one. Not sure if it'll work out though, but wanted to share it either way :)

**Comment by simonwep** [comment]
Closing this, as of v2 it's a single container :)
